### DISCUSSION TITLE
Support for 'text-wrap' ('pretty, 'balance')

### DISCUSSION THREAD

**Discussion by nathanchase** [csstools/postcss-plugins#1181] [Ideas]
I'd like to use PostCSS to use `text-wrap` in all browsers, but it doesn't appear there are any plugins or support for it in `postcss-preset-env` currently.

https://caniuse.com/?search=text-wrap
`text-wrap: balance`
`text-wrap: pretty`

https://developer.chrome.com/blog/css-text-wrap-balance/
https://developer.chrome.com/blog/css-text-wrap-pretty/

**Comment by romainmenke** [comment]
Hi @nathanchase,

Such a feature can not be polyfilled with PostCSS.

But luckily there is a JS polyfill you can use : https://github.com/adobe/balance-text
